packages/react-transport-dom-webpack/src/ReactFlightClientWebpackBundlerConfig.js

```diff
@@ -59,5 +59,16 @@ export function requireModule<T>(moduleData: ModuleReference<T>): T {
       throw entry;
     }
   }
-  return __webpack_require__(moduleData.id)[moduleData.name];
+  const moduleExports = __webpack_require__(moduleData.id);
+  if (moduleData.name === '*') {
+    // This is a placeholder value that represents that the caller imported this
+    // as a CommonJS module as is.
+    return moduleExports;
```

Comment:
I usually test CJS server paths by removing the `"type": "module"` definition in package.json. However I couldn't test the Webpack client paths for true CJS.

I spent a few hours trying to figure out how to make the react-script config let me write raw CJS instead of transpiling from ESM and I couldn't because webpack kept treating them as ESM which means that assigning to module.exports causes and error.